### PR TITLE
feat: make contact address and deployment hieght cli args

### PR DESCRIPTION
**Pull request by dexturr**
Allow custom deployment block height and contract address

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -38,17 +38,20 @@ const (
...
-	vestingContractAddress = common.HexToAddress("0x23d1bFE8fA50a167816fBD79D7932577c06011f4")
 	pollEventRetryDuration = 14 * time.Second // pretty much how long it takes to build a block
 	pollNewState           = time.Minute      // no need to poll too often
 )
 
 func init() {
 	flag.UintVar(&port, "port", 1789, "port of the http server")
 	flag.StringVar(&ethereumRPC, "eth-rpc", "", "ethereum RPC address")
+	flag.StringVar(&vestingContractAddress, "vesting-contract-address", "", "vesting contract address")
+	flag.StringVar(&deploymentHeightStr, "deployment-height", "", "deployment block height")
```
Comment (jeremyletang) [inline main.go]: How about setting defaults to mainnet values?

Comment (jeremyletang) [inline main.go]: You can make it a uint from the beginning as well, instead of a string that you have to parse later.

File: main.go [hunk 2]
```diff
@@ -447,6 +454,18 @@ func main() {
...
+		log.Fatal("error: missing vesting-contract-address parameter")
+	}
+
+	if len(deploymentHeightStr) <= 0 {
+		log.Fatal("error: missing deployment-height parameter")
+	}
+
+	if len(deploymentHeightStr) <= 0 {
+		log.Fatal("error: missing deployment-height parameter")
+	}
```
Comment (jeremyletang) [inline main.go]: Why check it twice?

Comment (dexturr) [reply main.go]: Removed entirely as there are now defaults

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**